src/contentScript.js

```diff
@@ -24,9 +24,53 @@ function findWrappingElement (element) {
  * @param {string} subreddit
  */
 async function handleMarkup (wrapperEl, subreddit) {
-	// TODO: make generic for more subreddits, for now I'm just hardcoding stuff for /r/anime
-	if (subreddit !== 'anime') return;
-	const commentFaces = wrapperEl.querySelectorAll('a[href^="#"],a[href="/s"]');
+	// TODO: make generic for more subreddits
+
+	const commentFaceSelector = [];
+	if (['anime',
+		'awwnime',
+		'pantsu',
+		'moescape',
+		'twodeeart',
+		'patchuu',
+		'kemonomimi',
+		'visualnovels',
+		'supersonico',
+		'kanmusu',
+		'kanmusunights',
+		'schoolidolfestival',
+		'lovelive',
+		'onetrueidol',
+		'fatestaynight',
+		'saber',
+		'karanokyoukai',
+		'nisekoi',
+		'onetruebiribiri',
+		'onetruetohsaka',
+		'esdeath',
+		'gamindustri',
+		'kancolle',
+		'leagueoflegends',
+		'chibi'].includes(subreddit.toLowerCase())) {
```

Comment:
my ESLint config is bad and doesn't handle autofixes for stuff like this well, sorry, can you update this condition to look like:

```js
if ([
    'anime',
    // ...
    'chibi',
].includes(...)) {
    // ...
}
```

Reply:
Hmmm, ESLint complains with https://eslint.org/docs/rules/array-bracket-newline, should I modify the eslint config or just leave the code as is?

Reply:
hmmmmm not worth it since I don't even have CI set up on this repo, let's just leave it as it is and whenever I get around to updating my personal config I'll fix it